src/test/java/com/flightstats/hub/channel/ChannelValidatorTest.java

```diff
@@ -31,7 +31,7 @@ public void setUp() throws Exception {
         channelService = mock(ChannelService.class);
         validator = new ChannelValidator(channelService);
         when(channelService.channelExists(any(String.class))).thenReturn(false);
-        HubProperties.setProperty("hub.protect.channels", "false");
+        PropertyLoader.getInstance().setProperty("hub.protect.channels", "false");
```

Comment:
It'll be nice when we can move away from this mutable stuff and maybe use mocking, since this will carry over between tests, I think.